app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -196,6 +196,62 @@
   border-bottom: 2px solid #aaa;
 }
 
+.panel-boundary-bottom {
+  padding-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+  border-bottom: 2px solid #aaa;
+}
+
+.panel-boundary-right {
+  border-right: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.